src/bin/pg_autoctl/cli_create_drop_node.c

```diff
@@ -242,7 +244,7 @@ cli_create_postgres_getopts(int argc, char **argv)
 
 	int optind =
 		cli_create_node_getopts(argc, argv, long_options,
-								"C:D:H:p:l:U:A:Sd:n:f:m:MRVvqhP:r:xsN",
+								"C:D:H:p:l:U:A:Sd:a:n:f:m:MRVvqhP:r:xsN",
```

Comment:
My understanding is that without this, `getopt_long` would fail to parse the long option too. If that's not the case then we could consider using an empty string here all the time...

Reply:
I believe it's not necessary, but I think doing this change everywhere fits better in another PR. A few we would probably want to keep though.